systest/daemon_interrupt.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eux
+
+DATADIR=$(mktemp -d)
+
+# Start the node in daemon mode in the background
+$NANO_NODE_EXE --daemon --network dev --data_path $DATADIR &
+NODE_PID=$!
+
+# Allow some time for the node to start up completely
+sleep 10
```

Comment:
In an ideal world, we would have a way to detect when the node is fully functional and all objects have been constructed. A log to mark the end of init and start stages.

Reply:
I don't like relying on log output. I believe this task is usually accomplished by providing a health check endpoint. Having one would be great, but it's too much hassle for this test case. Even if the node is still initializing, the interrupt should be queued and handled properly.